ui/site/css/tv/_zen.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../../round/css/constants';
+
+%zen {
+  .zen & {
+    display: none;
+  }
+}
+
+#friend_box,
+.round__side,
```

Comment:
We have to do this here and not in [`_side.scss`](https://github.com/lichess-org/lila/blob/0b21ec8da8a9028cf3e3b02aeeff43693cbc6449/ui/site/css/tv/_side.scss) because those styles are used in lichess.org/games and we only don't want zen mode to affect that page

Reply:
then maybe a new class is needed in the DOM to differenciate them?